kratos/tests/test_time_discretization.py

```diff
@@ -104,24 +98,57 @@ def test_BDF6(self):
         self.assertEqual(KM.GetMinimumBufferSize(bdf), 7)
 
     def test_Newmark(self):
+        # testing defaults
```

Comment:
Do we really need these in python? I guess it doesn't hurt, but we should avoid needless compilation.